src/command_modules/azure-cli-rdbms/azure/cli/command_modules/rdbms/_params.py

```diff
@@ -93,6 +94,25 @@ def _complex_params(command_group):
             c.argument('start_ip_address', options_list=['--start-ip-address'], help='The start IP address of the firewall rule. Must be IPv4 format. Use value \'0.0.0.0\' to represent all Azure-internal IP addresses.')
             c.argument('end_ip_address', options_list=['--end-ip-address'], help='The end IP address of the firewall rule. Must be IPv4 format. Use value \'0.0.0.0\' to represent all Azure-internal IP addresses.')
 
+    for scope in ['mysql server vnet-rule', 'postgres server vnet-rule']:
+        with self.argument_context(scope) as c:
+            c.argument('server_name', options_list=['--server-name', '-s'])
+            c.argument('virtual_network_rule_name', options_list=['--name', '-n'], id_part='child_name_1', help='The name of the vnet rule.')
+            c.argument('virtual_network_subnet_id', options_list=['--subnet'], help='Name or ID of the subnet that allows access to an Azure Postgres Server. If subnet name is provided, --vnet-name must be provided.')
+            c.argument('ignore_missing_vnet_service_endpoint', options_list=['--ignore-missing-endpoint', '-i'], help='Create vnet rule before virtual network has vnet service endpoint enabled', arg_type=get_three_state_flag())
```

Comment:
Is this a switch on the behavior of the command or a Boolean property on the rule resource?

Reply:
Can you please explain your question?  This will add new set of options for 
az postgres server/az mysql server  and ignore_missing_vnet_service_endpoint option lets the user specify true/false and assumes false by default.
.

Reply:
Is this a property of the rule object?

Reply:
yes , all these are properties of vnet-rule